src/walletSdk/horizon/AccountService.ts

```diff
@@ -0,0 +1,18 @@
+import { Keypair } from "stellar-sdk";
+
+import { SigningKeypair } from "./Account";
+
+// Do not create this object directly, use the Wallet class.
+export class AccountService {
+  private server;
+  private network;
+
+  constructor(cfg) {
+    this.server = cfg.stellar.server;
+    this.network = cfg.stellar.network;
+  }
+
+  createKeypair() {
+    return new SigningKeypair(Keypair.random());
```

Comment:
This won't work on react native. Not sure what's the best way of working it around, maybe adding typescript-wallet-sdk-react-native module later?
Reference: https://github.com/stellar/js-stellar-sdk#using-in-an-expo-managed-workflow

Reply:
Interesting - it looks like using `expo-random` to generate a keypair should work in the browser. Should we just default to that? It'd be great if we didn't have to branch logic for react native (not sure if there are other areas where we definitely have to do that)

Reply:
ah yeah, it looks like it doesn't run in node. Yeah, we could do a try with the native `crypto` lib for node and fallback to `expo-crypto` (that's actually the new lib, expo-random has been deprecated)

Reply:
IMO -- let's do it later. We should add some basic e2e tests for all platforms we want to support (browser, node, RN)

Reply:
I think this'll take some more digging into. I created a ticket to track here: https://stellarorg.atlassian.net/browse/WAL-885

going to leave for now if that's alright